draft-ietf-core-problem-details.md

```diff
@@ -588,7 +589,14 @@ Initial entries in this sub-registry are as follows:
 
 This specification defines a new sub-registry for Custom Problem
 Detail Keys in the CoRE Parameters registry {{!IANA.core-parameters}},
-with the policy "first come first served" {{!RFC8126}}.
+with the policy "expert review" ({{Section 4.5 of -ianacons}}).
+
+The expert is instructed to attempt making the registration experience
+as close to first-come-first-served as reasonable achievable, but
+checking that the reference document does provide a description as set
+out below.
+(This requirement is a relaxed version of "specification required" as
```

Comment:
Not sure this is necessary

Reply:
I wrote this mainly to fend of the knee-jerk "but this is like specification required, why don't you use that?".

Reply:
mmm, reminds me of St Jerome's letter 4: _"[...] dum excusare credis, accusas"_, but OK :-)